app/Http/Controllers/EventAttendeesController.php

```diff
@@ -662,7 +662,7 @@ public function postCancelAttendee(Request $request, $event_id, $attendee_id)
                 $message->to($attendee->email, $attendee->full_name)
                         ->from(config('attendize.outgoing_email_noreply'), $attendee->event->organiser->name)
                         ->replyTo($attendee->event->organiser->email, $attendee->event->organiser->name)
-                        ->subject('You\'r ticket has been cancelled');
+                        ->subject('Your ticket has been cancelled');
```

Comment:
You just have to do git rebase -i master and follow onscreen instructions

On Sun, 29 May 2016 13:23 JapSeyz, <notifications@github.com> wrote:

> In app/Http/Controllers/EventAttendeesController.php
> <https://github.com/Attendize/Attendize/pull/115#discussion_r65009272>:
>
> > @@ -662,7 +662,7 @@ public function postCancelAttendee(Request $request, $event_id, $attendee_id)
> >                  $message->to($attendee->email, $attendee->full_name)
> >                          ->from(config('attendize.outgoing_email_noreply'), $attendee->event->organiser->name)
> >                          ->replyTo($attendee->event->organiser->email, $attendee->event->organiser->name)
> > -                        ->subject('You\'r ticket has been cancelled');
> > +                        ->subject('Your ticket has been cancelled');
>
> I am not extremely experienced with Git, and will look up how to do that
> in the future.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/Attendize/Attendize/pull/115/files/f4b081650e76ef6452fa531ab23780bb87e06629..d6101b82867c1b6417c5235dd8878e086b7711d0#r65009272>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABx8OeQevN_TOJcMCHLztXXzuUFwNXisks5qGdmGgaJpZM4IpRqU>
> .
>